src/wallet/rpcwallet.cpp

```diff
@@ -2480,10 +2480,9 @@ Value zc_raw_receive(const json_spirit::Array& params, bool fHelp)
     uint256 commitment = uint256(commitment_v);
 
     assert(pwalletMain != NULL);
-    libsnark::merkle_authentication_path path(INCREMENTAL_MERKLE_TREE_DEPTH); // We don't care during receive... yet! :)
```

Comment:
Not in the case of receive and in the case of our payment API either, but good question.